### PR TITLE
Start nfs service for micro ltp test

### PR DESCRIPTION
**Pull request by coolgw**
nfs-server service is disabled by default in micro lead nfs related case such as statx07 failed.

- Related ticket: https://progress.opensuse.org/issues/161918
- Needles: na 
- Verification run: 
installation job: https://openqa.suse.de/tests/14810586#
case job: https://openqa.suse.de/tests/14810610#step/statx07/7
               https://openqa.suse.de/tests/14837330#step/statx07/7

NOTE: No vsftpd found in current sle-micro.  so make another workaround in code.
```
localhost:~ # zypper lr -u
Repository priorities are without effect. All enabled repositories share the same priority.

# | Alias                                                   | Name     | Enabled | GPG Check | Refresh | URI
--+---------------------------------------------------------+----------+---------+-----------+---------+--------------------------------------------------------------------------------
1 | SUSE_Linux_Micro_6.1_x86_64:SL-Micro-6.1-Debuginfo-Pool | SL-Mic-> | No      | ----      | ----    | http://openqa.suse.de/assets/repo/SL-Micro-Packages-6.1-x86_64-Debug-Build3.1/
2 | SUSE_Linux_Micro_6.1_x86_64:SL-Micro-6.1-Pool           | SL-Mic-> | Yes     | (r ) Yes  | No      | http://openqa.suse.de/assets/repo/SL-Micro-Packages-6.1-x86_64-Build3.1/
3 | SUSE_Linux_Micro_6.1_x86_64:SL-Micro-6.1-Source-Pool    | SL-Mic-> | No      | ----      | ----    | http://openqa.suse.de/assets/repo/SL-Micro-Packages-6.1-x86_64-Source-Build3.1/
4 | qa-head                                                 | qa-head  | Yes     | (  ) No   | Yes     | http://download.suse.de/ibs/QA:/Head/ALP-Standard-Core-1.0-Build/
localhost:~ # zypper se ftp
Refreshing service 'SUSE_Linux_Micro_6.1_x86_64'.
Loading repository data...
Reading installed packages...

S | Name             | Summary                               | Type
--+------------------+---------------------------------------+-----------
  | qa_test_ftpload  | ftp download test                     | package
  | qa_test_ftpload  | ftp download test                     | srcpackage
i | system-user-tftp | System user and group tftp            | package
i | tftp             | Trivial File Transfer Protocol (TFTP) | package

    Note: For an extended search including not yet activated remote resources please use 'zypper
    search-packages'.
```

### CODE REVIEW

File: tests/kernel/boot_ltp.pm [hunk 1]
```diff
@@ -58,6 +58,11 @@ sub run {
 
     log_versions;
 
+    # Enable nfs service for ltp test
+    if (is_sle_micro) {
+	    assert_script_run('systemctl restart nfs-server');
```
Comment (czerw) [inline tests/kernel/boot_ltp.pm:63]: It would be better to enable service in `install_ltp` to have image in final state.

Comment (czerw) [reply tests/kernel/boot_ltp.pm:63]: Moreover, we are skipping whole ` setup_network` function in install_ltp`. I think we should try it to run first to see what are the issues and adapt it for micro.

Comment (czerw) [reply tests/kernel/boot_ltp.pm:63]: https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/a1b8ef6dba68a1088dfba2c3d3330612fa91a1b5/tests/kernel/install_ltp.pm#L375

Comment (coolgw) [reply tests/kernel/boot_ltp.pm]: Following fail happen if you directly enable setup_network. So i do some update on this PR and also update VR in description.

https://openqa.suse.de/tests/14794807#step/install_ltp/319
touch: cannot touch '/var/lib/dhcp/db/dhcpd.leases': No such file or directory

https://openqa.suse.de/tests/14794825#step/install_ltp/319
sed: couldn't open temporary file /usr/etc/sedn4Ht0q: Read-only file system

File: tests/kernel/install_ltp.pm [hunk 2]
```diff
@@ -257,19 +258,24 @@ sub setup_network {
     # boo#1017616: missing link to ping6 in iputils >= s20150815
     assert_script_run('which ping6 >/dev/null 2>&1 || ln -s `which ping` /usr/local/bin/ping6');
 
-    # dhcpd
-    assert_script_run('touch /var/lib/dhcp/db/dhcpd.leases');
-    script_run('touch /var/lib/dhcp6/db/dhcpd6.leases');
+    if (!is_transactional) {
```
Comment (czerw) [inline tests/kernel/install_ltp.pm:261]: use `unless`

**Comment by czerw** [comment]
@coolgw please also do verification on older sle micro https://openqa.suse.de/tests/overview?distri=sle-micro&version=5.2&build=%3A34695%3Akernel-rt&groupid=368

**Comment by czerw** [comment]
@pevik @mdoucha please review too

**Comment by pevik** [comment]
You put a lot of info in PR description but no info in git commit message. Could you please extend it?

**Review by pevik — APPROVED** [review]
Apart from adding info to the commit message (+ optional adding more comments to the code) LGTM. Thanks!

**Review by mdoucha — APPROVED** [review]
LGTM

**Comment by coolgw** [comment]
> @coolgw please also do verification on older sle micro https://openqa.suse.de/tests/overview?distri=sle-micro&version=5.2&build=%3A34695%3Akernel-rt&groupid=368

Following strange error happen when i try to clone job with my branch, need further check:
https://openqa.suse.de/tests/14912483#step/update_kernel/122

**Comment by mdoucha** [comment]
> Following strange error happen when i try to clone job with my branch, need further check: https://openqa.suse.de/tests/14912483#step/update_kernel/122

This is expected. The kernel update was released and now it gets installed from SCC instead of from the incident repo. You'll have to try again with another kernel update or KOTD.

**Comment by coolgw** [comment]
> > Following strange error happen when i try to clone job with my branch, need further check: https://openqa.suse.de/tests/14912483#step/update_kernel/122
> 
> This is expected. The kernel update was released and now it gets installed from SCC instead of from the incident repo. You'll have to try again with another kernel update or KOTD.

https://progress.opensuse.org/issues/163886

**Comment by coolgw** [comment]
> > > Following strange error happen when i try to clone job with my branch, need further check: https://openqa.suse.de/tests/14912483#step/update_kernel/122
> > 
> > 
> > This is expected. The kernel update was released and now it gets installed from SCC instead of from the incident repo. You'll have to try again with another kernel update or KOTD.
> 
> https://progress.opensuse.org/issues/163886

Use micro_nfs_verify_debug private branch verify result:
install job: https://openqa.suse.de/tests/14991649#
Sub job result:
https://openqa.suse.de/tests/14991786
https://openqa.suse.de/tests/14991787
https://openqa.suse.de/tests/14991788
https://openqa.suse.de/tests/14991789
https://openqa.suse.de/tests/14991790
https://openqa.suse.de/tests/14991791
https://openqa.suse.de/tests/14991792
https://openqa.suse.de/tests/14991793
https://openqa.suse.de/tests/14991794
https://openqa.suse.de/tests/14991795

https://openqa.suse.de/tests/14991795 <<<< nfs still failed on some cases, need further investigation.

**Pull request merged**